docs/socket-groups.md

```diff
@@ -153,37 +175,47 @@ protection, the mechanism should quickly detect a link as broken so
 that packets lost on the broken link can be resent over the others,
 but no such mechanism has been provided for balancing group.
 
+Please also keep in mind that there's a rule for all groups - one member
+link established is enough for a group to be connection established. This
+might be not always wanted in case of balancing groups, so your application
+might need to do additional check and monitor how many links and which
+ones are currently active. For example, if you need 3 links to balance
+the load and having only 2 of them would be too little to withstand the
+whole transmission, you should wait with connecting until all 3 links
+are established. Important thing here is also that one of the links might
+be at some moment not possible to be established, or one of the links
+can get broken during transmission.
+
```

Comment:
```suggestion
Please also keep in mind that this rule applies to all groups: *establishing 
one member link is enough for a group connection to be established.* But 
when balancing groups, this might not always be wanted. In such cases, 
your application might need to monitor how many links are established, and 
which ones are currently active. For example, if you need a minimum of 3 links 
to balance the load, you should delay connecting until all 3 links are established. 
It is also important to take into consideration that it might not be possible to establish 
one of the links, or one of the links might get broken during transmission.

```

Reply:
Partially distributed in other parts; text to be kept copied from here.

Reply:
This has been distributed more in other places, so the text was mainly derived from this, but also changed.